template/php/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM php:7-alpine
```

Comment:
Is PHP7 being stuck with?  7.0 is EOL for bug fixes and into security only fixes now.

http://php.net/supported-versions.php

Reply:
7-alpine is the latest in the 7 branch, I can lock this to say 7.2 if that would be more appropriate otherwise it'll be the latest stable

Reply:
It shouldn't be a problem using the latest, but as PHP doesn't follow semver (can't find sources, have been informed of this in the past though), there could be issues if it just pulls the latest thing when the next `.X` release is out?